components/collapse/collapse.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component, PropTypes } from 'react';
+
+import CollapseItem from './collapseItem';
```

Comment:
accordion - it's behavior, in this case only one item can be expanded, I added props for that.